aten/src/ATen/core/LazyTensor.h

```diff
@@ -0,0 +1,57 @@
+#pragma once
+
+#include <ATen/core/TensorBody.h>
+#include <ATen/core/ivalue.h>
+#include <ATen/core/function_schema.h>
+#include <c10/core/TensorImpl.h>
+
+namespace at {
```

Comment:
Can we copy paste the design doc for lazy tensor somewhere? Maybe in `LazyTensor.cpp`?

Reply:
Or at the very least, a little bit of comments saying "this is a lazy tensor, this is what it represents" :)

Reply:
This comment is still not done

Reply:
It looks like the comments got put in `torch/csrc/LazyTensor.cpp` which is pretty far away from `aten/src/ATen/core/LazyTensor.{h,cpp}`. If you think `torch/csrc` is the right place to put the docs (I personally think the docs should be placed at the "lowest" spot where things are defined, aka `aten/src/ATen/core`) then at the very least there should be some cross-references pointing there, e.g., `See Note [Lazy tensor]`, for example